### DISCUSSION TITLE
where can i find  lotus client cli ?

### DISCUSSION THREAD

**Discussion by kyoungbinkim** [filecoin-project/lotus#12208] [General Q&A]
I'm developing along the lotus docs, but I can't find the `lotus client` command line.

dcos : https://lotus.filecoin.io/tutorials/lotus/build-with-lotus-api/#test-a-simple-function 

![스크린샷 2024-07-11 14-46-02](https://github.com/filecoin-project/lotus/assets/83156897/cda29882-371a-4cd2-ac5c-1b38cfd5c3b8)

**Comment by TippyFlitsUK** [comment]
https://github.com/filecoin-project/lotus/pull/11999

**Reply by kyoungbinkim** [reply]
thanks

**Reply by CLEILI** [reply]
Could you please tell me what I need to use to replace the functionality of lotus client CLI to upload or retrieve data?

**Reply by TippyFlitsUK** [reply]
https://boost.filecoin.io/
